uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -666,7 +667,11 @@ define([
 
             self.postContactSelection = false;
           })
-          .catch(handleError.bind(self));
+          .catch(function (error) {
```

Comment:
It is handled in the parent function(from where `initAfterContactSelection` is called) i.e. `_init`.
```javascript
then(function () {
  return self.initAfterContactSelection();
})
.catch(handleError.bind(self));
```